src/main/java/Duke.java

```diff
@@ -1,10 +1,228 @@
+                    String[] ToSplitTodo = line.split(" ");
+                    try {
+
+                        String TodoTask = line.toLowerCase().replaceAll("todo","");
+                        tasks.add(new Todo(TodoTask,ToSplitTodo.length));
+
+
+                    } catch (DukeException ex) {
+                        empty = true;
+                        System.out.println("____________________________________________________________\n"
+                                + "OOPS!!! The description of a todo cannot be empty.\n"
+                                + "____________________________________________________________\n");
+                    }
+
+                    //sample : deadline return book /by Sunday
+                } else if (line.toLowerCase().contains("deadline")) {
+                    String[] ToSplitDeadline = line.split("/");
+                    String DeadlineTask = ToSplitDeadline[0].toLowerCase().substring(9, ToSplitDeadline[0].length() - 1);
+                    tasks.add(new Deadline(DeadlineTask, ToSplitDeadline[1].substring(3, ToSplitDeadline[1].length())));
+
+                    //sample:event project meeting /from Mon 2pm /to 4pm
+                } else if (line.toLowerCase().contains("event")) {
+                    String[] ToSplitEvent = line.split("/");
+                    String EventTask = ToSplitEvent[0].toLowerCase().substring(6, ToSplitEvent[0].length());
+                    tasks.add(new Event(EventTask, ToSplitEvent[1].substring(5, ToSplitEvent[1].length()), ToSplitEvent[2].substring(3, ToSplitEvent[2].length())));
+
+                }
+
+                if(!empty) {
+                    System.out.println("____________________________________________________________\n"
+                            +"Got it. I've added this task:\n" +
+                            "  " + tasks.get(i) + "\n" +
+                            "Now you have " + (i + 1) + " tasks in the list.\n"
+                            + "____________________________________________________________\n");
+                    i += 1;
+
+                    try {
+                        appendToFile("Duke.txt",  tasks.get(i - 1).toString() + "\n");
+                    } catch (IOException e) {
+                        System.out.println("Something went wrong\n"
+                                + "____________________________________________________________");
+                    }
+                }
+
+            } else if (line.equalsIgnoreCase("list")) {
+
+                System.out.println("____________________________________________________________\n" +
+                        "Here are the tasks in your list:");
+
+                for (int m = 0; m < i; m += 1) {
+                    int index = m + 1;
+
+                    System.out.println(index + "." + tasks.get(m));
+
+                }
+                System.out.println("____________________________________________________________");
```

Comment:
You can consider using constants to store the "________________________________________" string literal. It will improve readability by eliminating the magic literals.